File: flax_search_service/docs/tutorial/python/t1_create_db.py
import flax.searchclient

# open a connection to Flax Server
conn = flax.searchclient.Client('http://localhost:8080')

# create a database, overwriting any existing db with the same name
conn.create_database('books', overwrite=True)

# create a database schema
schema = conn.schema('books')

# add some field definitions
schema.add_field('title', {'freetext': {'language': 'en'}, 'store': True})
schema.add_field('first', {'freetext': {'language': 'en'}, 'store': False})
schema.add_field('author', {'exacttext': True, 'store': True})

File: flax_search_service/docs/tutorial/php/t1_create_db.php
<?php
include_once "flax/flaxclient.php";

# open a connection to Flax Server
$conn = new FlaxSearchService('http://localhost:8080');

# create a database, overwriting any existing db with the same name
$db = $conn->createDatabase('books', true);

# add some field definitions
$db->addField('title', new FlaxFreeTextField(true, 'en'));
$db->addField('first', new FlaxFreeTextField(false, 'en'));
$db->addField('author', new FlaxExactTextField(true));

?>